app/routes/api/claim.js

```diff
@@ -223,6 +223,41 @@ const addHerdToPreviousClaims = async (herdClaimData, applicationReference, sbi,
   ))
 }
 
+const createClaim = async ({ isMultiHerdsClaim, logger, amount, payload, claimReference, statusId, sbi }) => {
+  let herdGotUpdated = false
+  let herdData = {}
+  const { createdBy, applicationReference } = payload
+  const { herd, ...payloadData } = payload.data
+  const { typeOfLivestock } = payloadData
```

Comment:
`const {
  createdBy,
  applicationReference,
  data: { herd, ...payloadData }
} = payload;

const { typeOfLivestock } = payloadData;`

this might be a little simpler, saves you destructuring quite as much